.projenrc.js

```diff
@@ -21,7 +21,7 @@ const project = new Cdk8sTeamNodeProject({
     'cdk8s-plus-26',
     'cdk8s-plus-27',
     'constructs',
-    'lerna',
+    'lerna@^7.1.4',
```

Comment:
No need, this will prevent minor version upgrades, which we do want...

```suggestion
    'lerna',
```